grails-app/services/org/pih/warehouse/core/DashboardService.groovy

```diff
@@ -554,6 +554,50 @@ class DashboardService {
         return reorderStock
     }
 
+    def getReorderReport(Location location) {
+        long startTime = System.currentTimeMillis()
+        ArrayList<Map> inventoryItems = getInventoryItems(location)
+
+        ArrayList<Map> reorderStock = inventoryItems.findAll { Map inventoryItem ->
+            def inventoryLevel = inventoryItem?.inventoryLevel as InventoryLevel
+
+            if (inventoryLevel?.binLocation != null) {
```

Comment:
Add a comment (for future you's sake) to explain why we're doing this. It's probably not obvious to a develop coming in cold.